android/library/src/main/java/io/logto/android/LogtoConfig.kt

```diff
@@ -0,0 +1,20 @@
+package io.logto.android
+
+data class LogtoConfig(
+    val clientId: String,
+    val oidcEndpoint: String,
+    val scopes: List<String>,
+    val redirectUri: String,
+) {
+
+    fun getEncodedScopes(): String = scopes.joinToString(" ")
```

Comment:
let's use [getters and setters](https://kotlinlang.org/docs/properties.html#getters-and-setters) instead of java-style functions.